tools/building.py

```diff
@@ -735,21 +736,6 @@ def metadce(js_file, wasm_file, minify_whitespace, debug_info):
       export = asmjs_mangle(item['export'])
       if settings.EXPORT_ALL or export in required_symbols:
         item['root'] = True
-  # in standalone wasm, always export the memory
-  if not settings.IMPORTED_MEMORY:
-    graph.append({
-      'export': 'memory',
-      'name': 'emcc$export$memory',
-      'reaches': [],
-      'root': True
-    })
-  if not settings.RELOCATABLE:
-    graph.append({
-      'export': '__indirect_function_table',
-      'name': 'emcc$export$__indirect_function_table',
-      'reaches': [],
-      'root': True
-    })
```

Comment:
Why are these no longer needed?

Reply:
I mention this is in the PR description.     Basically we now take into account all exports in in the metadce graph generation so we don't need these hard-coded special cases anymore.   Previously we were only passing function and tag exports to emitDCEGraph.